deploy/helm/lunettes/templates/grafanadi/grafanadi-deploy.yaml

```diff
@@ -20,9 +20,25 @@ spec:
           - 0
       containers:
       - command:
-        - ./grafanadi
-        - --config-file=/app/config-file.yaml
-        - --enable-kube-client={{ .Values.grafanadiInitKubeClient }}
+        - /bin/sh
+        - -c
+        - |
+          echo "Waiting for elastic search cluster to start..."
+          status_code="000X"
+          while true;
+          do
+            if [ ${status_code} = "200X" ];
+            then
+              echo "Elastic search cluster start successfully!"
+              break
+            fi
+            status_code=`curl -s --connect-timeout 3 --max-time 5 \
```

Comment:
you can ref https://github.com/alipay/container-observability-service/blob/e955cefb2ba9ffb21f5e0c65f937a6cf044be146/.github/workflows/e2e-test.yml#L58-L65 to check es is ready or not